src/aiida/orm/nodes/data/dict.py

```diff
@@ -51,15 +50,12 @@ class Dict(Data):
     Finally, all dictionary mutations will be forbidden once the node is stored.
     """
 
-    __qb_fields__ = [
-        add_field(
-            'dict',
-            dtype=DictType[str, Any],
+    class Model(Data.Model):
+        value: t.Dict[str, t.Any] = MetadataField(
+            description='Dictionary content.',
```

Comment:
Isn't this a breaking change for the query builder? I guess it is not horrible but at least we have to note this down somewhere. I feel like making a deprecation warning would be quite hard.

Reply:
Not sure to be honest. This qbfields concept was only added in v2.6 I believe. I am not sure how the `dict` field would have been used in generic queries for `Data` nodes. I don't think you would actually use it in projections, because `dict` is just an alias for the entire `attributes` column of the `DbNode` table. So if you want to project this, you would do `*` anyway. Same goes for filtering, not sure how you would use it. But I have been out of it too long to remember by heart. I would recommend to check how it worked in v2.6 and see if that still works here.